SPECS/sysstat/sysstat.spec

```
@@ -1,13 +1,14 @@
 Summary:        The Sysstat package contains utilities to monitor system performance and usage activity
 Name:           sysstat
 Version:        12.7.1
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        GPLv2
```

Mind updating this to use an [SPDX license expression](https://spdx.org/licenses/), while you're here?

```suggestion
License:        GPL-2.0-only
```